docs/template_plugin/tests/functional/subgraph_reference/preprocess_legacy.cpp

```diff
@@ -96,11 +100,11 @@ TEST_F(ReferencePreprocessLegacyTest, resize) {
     auto f2 = create_simple_function(element::f32, Shape{1, 3, 5, 5});
     legacy_network = InferenceEngine::CNNNetwork(f2);
 
-    function = PrePostProcessor(function).input(InputInfo()
-            .tensor(InputTensorInfo().set_layout("NCHW").set_spatial_static_shape(42, 30))
-            .preprocess(PreProcessSteps().resize(ResizeAlgorithm::RESIZE_LINEAR))
-            .network(InputNetworkInfo().set_layout("NCHW")))
-                    .build();
+    auto p = PrePostProcessor(function);
+    p.input().tensor().set_layout("NCHW").set_spatial_static_shape(42, 30);
+    p.input().preprocess().resize(ResizeAlgorithm::RESIZE_LINEAR);
+    p.input().network().set_layout("NCHW");
+    p.build();
```

Comment:
result of `build()` is not used here, is it OK?

Reply:
'build' modifies original 'function', it doesn't create a copy, so it is ok to use it this way